class-metamodel/class-and-attribute-subsystem/class-attribute.xmm

```diff
@@ -109,7 +110,7 @@ relationships
     R21
     constrains value of, Mc Attribute
     is constrained to match value of, Mc Identifier Attribute
-    1 Attribute Reference
+    M Attribute Reference
```

Comment:
There could be multiple representations of “Attribute Reference” between the two classes related by the R21 association. Possible solution change multiplicity to many on the association class of the association. Example of this could be found at the Shlaer-Mellor Metamodel, Relationship Subsystem for attribute references caused by R117, R118.

Reply:
Thank you!!! I remember puzzling over the attributes in the Attribute Reference class and the identifier just didn't look right to me. I knew something weird was going on, but couldn't think of the right example at the time. R117,R118 in the Reference Subsystem is a perfect example.  I am writing this up in the R21 relationship description!